globus_sdk/auth/token_response.py

```diff
@@ -30,6 +33,7 @@ def __init__(self, *args, **kwargs):
         GlobusHTTPResponse.__init__(self, *args, **kwargs)
         # dict used to avoid re-computing self.by_resource_server
         self._by_resource_server = None
+        self._by_resource_server = self.by_resource_server
```

Comment:
Sorry, you're right. I was going to do it in the dependent token class but forgot.